gulpfile.mjs

```diff
@@ -2344,6 +2312,18 @@ gulp.task(
   })
 );
 
+gulp.task(
+  "typestest",
+  gulp.series(setTestEnv, "dist", function runTypesTest(done) {
+    exec('"node_modules/.bin/tsc" -p test/types', function (err, stdout) {
+      if (err) {
+        console.log(`Couldn't compile TypeScript test: ${stdout}`);
+      }
+      done(err);
+    });
+  })
+);
+
```

Comment:
Why was this code-block moved, since that seems unrelated and just increases the size of the patch?